teams/vsp/product-management/product-brief-template.md

```diff
@@ -1,31 +1,36 @@
 # **VSP Product Brief Template**
 
-### What is it?
+To create a Product Brief:
```

Comment:
```suggestion
Use this template to create a Product Brief for your product. To begin,
```

Perhaps we should mention something about up-keep, i.e. "ensure changes are tracked"

Reply:
-1 to "ensure changes are tracked", it's true but telling people at the start, "remember to come back to this at some unknown point in the future" isn't very usefully actionable.

instead, I lean on things like Initiative Dev Checklist as the forcing function for "revisit your product / user guide docs."

accepting your wording suggestion though :+1: